CI-API/Controllers/LandingPageController.cs

```diff
@@ -92,13 +92,13 @@ public async Task<JsonResult> RecommendedMissionCall(long MissionId, long FromUs
 
         #region
         [HttpPost("RecommendedUserGet")]
-        public async Task<JsonResult> RecommendedUsers(long MissionId, long LoginUserId)
+        public async Task<JsonResult> RecommendedUsers(long MissionId, long UserId)
```

Comment:
If this is a data retrieval than use **HttpGet** instead

Reply:
okay sir...done this one change